src/addrman.h

```diff
@@ -732,19 +732,15 @@ class CAddrMan
     void Check() const
         EXCLUSIVE_LOCKS_REQUIRED(cs)
     {
-#ifdef DEBUG_ADDRMAN
         AssertLockHeld(cs);
         const int err = Check_();
         if (err) {
             LogPrintf("ADDRMAN CONSISTENCY CHECK FAILED!!! err=%i\n", err);
         }
-#endif
     }
 
-#ifdef DEBUG_ADDRMAN
     //! Perform consistency check. Returns an error code or zero.
     int Check_() const EXCLUSIVE_LOCKS_REQUIRED(cs);
-#endif
```

Comment:
This patch builds and runs for me. Note, it completely removes the `#ifdef DEBUG_ADDRMAN` checks from `src/addrman.h`.